src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -216,6 +220,16 @@ public ThingInfo createThing(IotClient client, String policyName, String thingNa
                     .build());
         }
 
+        // handle endpoints
+        if (iotDataEndpoint == null || iotDataEndpoint.isEmpty()) {
```

Comment:
Nit: Use `Utils.isEmpty` instead.

Reply:
Nit: With this change, if customer's set this endpoint to a non-empty invalid value, we won't use and update it to the correct ones as we used to do before.

Reply:
Updated